tests/mocha/index.html

```diff
@@ -129,20 +129,18 @@
       <category name="First" expanded="true" categorystyle="logic_category">
         <sep gap="-1"></sep>
         <button text="insert" callbackkey="insertConnectionRows"></button>
-        <block type="basic_block">
-          <field name="TEXT">FirstCategory-FirstBlock</field>
-        </block>
-        <block type="basic_block">
-          <field name="TEXT">FirstCategory-SecondBlock</field>
-        </block>
+        <block type="stack_block"></block>
+        <block type="stack_block"></block>
       </category>
       <category name="Second">
-        <block type="basic_block">
-          <field name="TEXT">SecondCategory-FirstBlock</field>
-        </block>
+        <block type="stack_block"></block>
```

Comment:
Why do you change this? It seems like it would be better to be able to differentiate between the blocks.

Reply:
I don't know how much value it was adding, sounds fine to me then.